testground/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#:::
+#::: BUILD CONTAINER
+#:::
+
+# GO_VERSION is the golang version this image will be built against.
+ARG GO_VERSION=1.16
+
+# Dynamically select the golang version.
+FROM golang:${GO_VERSION}-buster
+
+# Testground version
+COPY VERSION /VERSION
```

Comment:
This is supposed to be passed to `docker build` as a build arg, I think the Ruby logic already does that
You can check existing images/builds for that